code/web/src/modules/user/Profile.js

```diff
@@ -24,23 +26,67 @@ const Profile = (props) => (
     </Helmet>
 
     {/* Top title bar */}
-    <Grid style={{ backgroundColor: grey }}>
-      <GridCell style={{ padding: '2em', textAlign: 'center' }}>
+    <Grid style={{ backgroundColor: grey }} justifyRight={true}>
+      <GridCell style={{ padding: '2em', textAlign: 'center', maxWidth: '20vw', marginRight: '24%' }}>
         <H3 font="secondary">My profile</H3>
       </GridCell>
+      
+      <GridCell style={{ padding: '2em', textAlign: 'center', maxWidth: '9vw' }}>
+        <Button theme="secondary" onClick={props.logout} style={{ marginLeft: '1em' }}>Logout</Button>
+      </GridCell>
     </Grid>
 
     <Grid>
       <GridCell style={{ padding: '2em', textAlign: 'center' }}>
-        <H4 style={{ marginBottom: '0.5em' }}>{props.user.details.name}</H4>
+        <img
+          src={`${APP_URL}//images/stock/men/4.jpg`}
```

Comment:
Glad we were able to figure out how to get images to show up! Hopefully as we add the ability for a user to upload an image, it shouldn't be too difficult to make this dynamic